packages/backend/src/graphql/resolvers/outgoing_payment.ts

```diff
@@ -60,6 +60,38 @@ export const createOutgoingPayment: MutationResolvers<ApolloContext>['createOutg
       }))
   }
 
+export const createOutgoingPaymentFromIncomingPayment: MutationResolvers<ApolloContext>['createOutgoingPaymentFromIncomingPayment'] =
+  async (
+    parent,
+    args,
+    ctx
+  ): Promise<ResolversTypes['OutgoingPaymentResponse']> => {
+    const outgoingPaymentService = await ctx.container.use(
+      'outgoingPaymentService'
+    )
+    return outgoingPaymentService
+      .create(args.input)
+      .then((paymentOrErr: OutgoingPayment | OutgoingPaymentError) => {
+        console.log('paymentOrErr', { input: args.input, paymentOrErr })
+        return isOutgoingPaymentError(paymentOrErr)
+          ? {
+              code: errorToCode[paymentOrErr].toString(),
+              success: false,
+              message: errorToMessage[paymentOrErr]
+            }
+          : {
+              code: '200',
```

Comment:
should probably be 201, but we are getting rid of code anyway

Reply:
fair point, probably shouldn't rely on those other changes. I updated this and the `createOutgoingPayment` resolver to be 201.